src/Features/CSharp/Portable/CodeRefactorings/ConvertToRecord/CSharpConvertToRecordRefactoringProvider.cs

```diff
@@ -146,6 +145,24 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 semicolon = default;
             }
 
+            // delete IEquatable if it's explicit because it is implicit on records
+            var iEquatable = ConvertToRecordHelpers.GetIEquatableType(semanticModel.Compilation, originalType);
+            var baseList = originalDeclarationNode.BaseList;
+            if (baseList != null && iEquatable != null)
+            {
+                var typeList = baseList.Types.Where(baseItem
+                    => iEquatable.Equals(semanticModel.GetSymbolInfo(baseItem, cancellationToken).Symbol));
```

Comment:
Is this missing a `!`? Because otherwise TypeList will contain the things that _are_ IEquatable, right? I suspect all the tests are passing because this is just removing all interfaces period -- do we have a test where one stays around?